### PR TITLE
Added API docs for GetUserTermsOfService API

### PR DESCRIPTION
**Pull request by harshilsharma**
Added API docs for GetUserTermsOfService API

### CODE REVIEW

File: v4/source/users.yaml [hunk 1]
```diff
@@ -1541,3 +1542,31 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+    get:
+      tags:
+      - users
+      - terms of service
+      summary: Fetches user's latest terms of service action if accepted.
```
Comment (grundleborg) [inline v4/source/users.yaml]: I don't really understand this. Does it mean that it fetches the latest "accept" action of the terms of service by a user?

File: v4/source/users.yaml [hunk 2]
```diff
@@ -1541,3 +1542,31 @@
...
+      description: |
+        Fetches user's latest terms of service action if accepted.
```
Comment (grundleborg) [inline v4/source/users.yaml]: As above.

File: v4/source/definitions.yaml [hunk 3]
```diff
@@ -2219,6 +2219,21 @@ definitions:
...
+    type: object
+    properties:
+      user_id:
+        type: string
+        description: The unique identifier of the user who performed this terms of service action.
+      terms_of_service_id:
+        type: string
+        description: The unique identifier of the terms of service the action was performed on.
+      create_at:
+        description: The time in milliseconds the user performed action on this terms of service.
```
[the remark this answers is not in the document]
Comment (harshilsharma) [reply v4/source/definitions.yaml]: Agreed. Done.

File: v4/source/users.yaml (hunk 1 above)
Comment (harshilsharma) [reply v4/source/users.yaml]: We only store a user's latest action if it was for accepting the TOS, and delete it on rejection. So the API only returns the accepted action, if the latest action was for acceptance.

Perhaps the updated text makes sense?

File: v4/source/users.yaml (hunk 2 above)
Comment (harshilsharma) [reply v4/source/users.yaml]: Updated.

**Comment by harshilsharma** [comment]
@hanzei I've created server PR with those docs - https://github.com/mattermost/mattermost-server/pull/9872

File: v4/source/users.yaml [hunk 4]
```diff
@@ -1556,3 +1557,33 @@
           $ref: '#/responses/Unauthorized'
         '403':
           $ref: '#/responses/Forbidden'
+    get:
+      tags:
+      - users
+      - terms of service
+      summary: Fetches user's latest terms of service action if the latest action for for acceptance.
```
Comment (grundleborg) [inline v4/source/users.yaml]: That feels a bit clearer to me now. Just a little typo to fix now (and below in the description). `for for acceptance` should probably be `was for acceptance`?

Comment (harshilsharma63) [reply v4/source/users.yaml]: Yes, that's a typo. Fixed now.

**Pull request merged**